pandas/tests/tseries/offsets/test_offsets.py

```diff
@@ -551,28 +563,144 @@ def test_mul(self):
         assert DateOffset(2) == 2 * DateOffset(1)
         assert DateOffset(2) == DateOffset(1) * 2
 
-    def test_constructor(self):
-
-        assert (self.d + DateOffset(months=2)) == datetime(2008, 3, 2)
-        assert (self.d - DateOffset(months=2)) == datetime(2007, 11, 2)
+    @pytest.mark.parametrize("relativedelta_kwd", list(liboffsets._relativedelta_kwds))
```

Comment:
This needs to be `list` -> `sorted` to prevent pytest collection errors with pytest-xdist.

Also, for consistency with elsewhere in this module, could maybe also change "relativedelta_kwd" -> "kwd"